ksql-common/src/test/java/io/confluent/ksql/metrics/MetricCollectorsTest.java

```diff
@@ -60,6 +67,16 @@ public void shouldAggregateStats() {
     assertThat(aggregateMetrics.values().iterator().next().getValue(), equalTo(3.0));
   }
 
+  @Test
+  public void shouldAddConfigurableReporters() {
+    assertThat(MetricCollectors.getMetrics().reporters().size(), equalTo(1));
+    when(ksqlConfig.getConfiguredInstances(any(), any()))
+        .thenReturn(Collections.singletonList(mock(MetricsReporter.class)));
+
+    MetricCollectors.addConfigurableReporter(ksqlConfig);
+    assertThat(MetricCollectors.getMetrics().reporters().size(), equalTo(2));
```

Comment:
cant we assert that one of the reporters is the mock that's returned by `getConfiguredInstances`?

Reply:
done